fastlane/Fastfile

```diff
@@ -21,8 +21,9 @@ end
 
 def clear_bundle_version
   puts "removing CFBundleVersion..."
-  update_plist(
-    plist_path: "./ios/Exponent/Supporting/Info.plist",
+  update_info_plist(
```

Comment:
Just curious, why change this to `update_info_plist`? The `xcodeproj` and `scheme` params don't seem to bear relevance to the work that this task does.

Reply:
The method seemed a better fit for the task, since the file being edited is an info.plist, and the two arguments uniquely identify it.  But I can't make a real case for the change, so let's skip it for now.

Reply:
Oh wait, I remember, it's because I saw this error locally and in CI:
```
[13:04:34]: Could not find action, lane or variable 'update_plist'. Check out the documentation for more details: https://docs.fastlane.tools/actions
```
Apparently, that action was accidentally deleted for a few fastlane releases.  It came back, but is there a reason not to use the other method?